raiden/api/rest.py

```diff
@@ -1239,7 +1242,7 @@ def _close(
             self.raiden_api.channel_close(
                 registry_address, channel_state.token_address, channel_state.partner_state.address
             )
-        except InsufficientFunds as e:
+        except (InsufficientEth, InsufficientFunds) as e:
```

Comment:
Same as above.

Reply:
There is already an integration test (`raiden/tests/integration/api/test_restapi.py::test_api_close_insufficient_eth`) that confirms that the exception is bubbled up in this case. It doesn't seem to work in the above case with `_withdraw` though.